src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.video.unit.spec.jsx

```diff
@@ -934,6 +934,27 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
         isHomepageRefresh: true,
       });
 
+      const facility = {
+        id: 'vha_442',
+        attributes: {
+          ...getVAFacilityMock().attributes,
+          uniqueId: '442',
+          name: 'Cheyenne VA Medical Center',
+          address: {
+            physical: {
+              zip: '82001-5356',
+              city: 'Cheyenne',
+              state: 'WY',
+              address1: '2360 East Pershing Boulevard',
+            },
+          },
+          phone: {
+            main: '307-778-7550',
+          },
+        },
+      };
+      mockFacilitiesFetch('vha_442', [facility]);
```

Comment:
I guess I'm confused by the purpose of this PR. I thought we (as in the VAOS team and other teams) already knew why these 404s were being generated, and that adding mocks was one way to get rid of them?

Are you asking us to update all the tests where this is happening? That will make our tests more verbose and introduce mocking that doesn't impact whether the test passes or fails. Plus, we often use the absence of a mock as a way to test that we correctly handle when an api request fails.

The "Error: 404" errors are custom messages that [I added in the platform code](https://github.com/department-of-veterans-affairs/vets-website/pull/13541/files#diff-46a1eb82499887b78f81ffaf26d07bfcc78a71164c47a8468812b3256bb12b91R44), how and when they're shown is fully under VSP's control. When they were added, they were meant as a debugging aid that you could opt into seeing, but a subsequent platform change exposed them by default in the local unit test output.

Reply:
> I guess I'm confused by the purpose of this PR. I thought we (as in the VAOS team and other teams) already knew why these 404s were being generated, and that adding mocks was one way to get rid of them?

VSP was not aware that these 404s were being intentionally generated until recently. After some discussion we've determine the best course of action is to have VAOS team address the unit tests printing 404s to the console at their convenience and discretion. The ultimate goal is to clean up the console output to make triaging easier as well as local development.

Let me know how you would prefer we proceed regarding this PR. We can close this if it's unnecessary as the purpose was to provide an example "fix" when we were operating under the assumption that developers were unaware what was causing these errors.